lib/relay/protocol/packets_xdr.go

```diff
@@ -31,7 +31,7 @@ struct header {
 
 */
 
-func (header) XDRSize() int {
+func (o header) XDRSize() int {
```

Comment:
Is this autogenerated?

Reply:
yes, as per comment in [packets.go](https://github.com/entity0xfe/syncthing/blob/relay_check_token/lib/relay/protocol/packets.go#L3-L4):  `go run github.com/calmh/xdr/cmd/genxdr  -o packets_xdr.go packets.go`.